src/main/java/com/wire/kalium/cli/CliSendApplication.kt

```diff
@@ -148,30 +136,31 @@ class CliApplication() : CliktCommand() {
         val content = msg.createGenericMsg().toByteArray()
         val encryptedMessage = crypto.encrypt(recipients, content)
         val param = MessageApi.Parameters.DefaultParameters(
-            sender = clientId,
-            priority = MessagePriority.LOW,
-            nativePush = false,
-            recipients = encryptedMessage,
-            transient = false,
+                sender = clientId,
+                priority = MessagePriority.LOW,
+                nativePush = false,
+                recipients = encryptedMessage,
+                transient = false,
```

Comment:
Weird indentation again? 🤔

Reply:
Hmm 🤔 weird indeed.

Did you run Command + L or is it leftover from before the restyle?

Reply:
fixed, i did run the Command + l and it fixed it 
i think it's from before the restyle

Reply:
i will do a rebase before merging just to make sure it's all good